focus-areas/people/contributors.md

```diff
@@ -1,75 +1,98 @@
+# **Contributors**
+
+**Question:** Who are the contributors to a project?
+
+## **Overview**
+A contributor is defined as anyone who contributes to the project in any way. This metric ensures that all types of contributions are fully recognized within the project. Identifying contributors helps the community recognize and celebrate various contributions, from code development to event planning and marketing efforts.
+
+Recognizing the contributors is vital to understanding who is driving different activities and informs project health by showing diversity and breadth in project involvement.
+
+## **Want to Know More?**
+
+<span markdown="1"><details>
+<summary>Click to read more about this metric.</summary>
+
+### **Data Collection Strategies**
+- **Aggregators:** Collect contributor names from collaboration tools, such as source code repositories, issue trackers, event registrations, mailing lists or any other tools a project uses.
+   - *Count:* Total number of contributors during a given time period.
+- **Parameters:**
+   - *Period of time:* Start and end dates for the period in which contributions are counted (default: forever).
+- **Data Sources:** Contributor data can be collected from collaboration platforms like GitHub, GitLab, IRC, blogs, and forums, or from tools like GrimoireLab and Augur.
+
+### **Filters**
+Contribution data can be filtered in several ways, such as:
+- By the location of engagement:
+   - Commit authors
+   - Issue authors
+   - Review participants (e.g., pull requests)
+   - Mailing list authors
+   - Event participants
+   - IRC or forum authors
+- By project characteristics:
+   - By release cycle
+   - Programming languages of the project
+   - Role or function in the project (e.g., coding, organizing, mentoring)
+   - Timeframe of activity (e.g., identifying new contributors)
+
+### **Visualizations**
+1. **List of Contributors:**  
+   Display contributor names with engagement details.
+<img width="490" alt="IMG_5268" src="https://github.com/user-attachments/assets/456864b5-6632-4fda-93a6-0ada9ca10b45">
+ 
+  *Figure 1: List of contributors and engagement (Source, Year)*
+
+2. **Summary of Contributors:**  
+   A summary showing the total number of contributors.
+![IMG_5269](https://github.com/user-attachments/assets/1d60591c-3024-4068-82ed-241846c03ac7)
+
+   *Figure 2: Summary number of contributors (Source, Year)*
+
+4. **Growth Over Time:**  
+   Change in the number of active contributors over time.
+![IMG_5270](https://github.com/user-attachments/assets/c13de6dc-8f08-4d5d-b98b-dba0b3673dc8)
+
+    *Figure 3: Contributor growth over time (Source, Year)*
+
+6. **New Contributors:**  
+   A list sorted by the date of first contribution.
+   
+![IMG_5271](https://github.com/user-attachments/assets/22f02ff1-6023-4ec1-92a9-a7efdeb2e25f) 
+  
+   *Figure 4: List of new contributors (Source, Year)*
+
+### **Surveys and Interviews**
```

Comment:
@yigakpoa Please make sure headers match template. Thanks!

Reply:
Actually can you move lines 63-75 up to under "Data Collection Strategies" and make this a subheader (as in, not on the same level as the Data Collection strategies"

Reply:
Thank you for pointing that out. I corrected it and moved it up